grpc/server/src/engine_server/mappings/ipc/step.rs

```diff
@@ -0,0 +1,142 @@
+use std::convert::{TryFrom, TryInto};
+
+use casper_execution_engine::core::engine_state::step::{RewardItem, SlashItem, StepRequest};
+use casper_types::{bytesrepr, PublicKey, U512};
+
+use crate::engine_server::{
+    ipc,
+    mappings::{MappingError, ParsingError},
+};
+use casper_types::bytesrepr::ToBytes;
+
+impl TryFrom<ipc::SlashItem> for SlashItem {
+    type Error = MappingError;
+
+    fn try_from(pb_slash_item: ipc::SlashItem) -> Result<Self, Self::Error> {
+        let bytes: Vec<u8> = pb_slash_item
+            .get_validator_id()
+            .try_into()
+            .map_err(|_| MappingError::Parsing(ParsingError("validator_id".to_string())))?;
```

Comment:
I'd like to see `const`s for these strings (and below)

Reply:
did this